### ISSUE TITLE
使用 OpenCore 1.0.2 数据库时，保存后验证有错误

### ISSUE THREAD

**Issue by beyondgary** [ic005k/OCAuxiliaryTools#409]
当保存后，OCA 会将 UEFI - Unload 节点删除，然后验证提示：
```
NOTE: This version of ocvalidate is only compatible with OpenCore version 1.0.2!

OCS: Missing key Unload, context <UEFI>!
Serialisation returns 1 error!

Completed validating /Volumes/EFI/efi/oc/config.plist in 1 ms. Found 1 issue requiring attention.
```

**Comment by ic005k** [comment]
请参阅：

https://github.com/ic005k/OCAuxiliaryTools/issues/407

**Comment by 5T33Z0** [comment]
> 请参阅：
> 
> #407

Are you planning to update the app or did you abandon it?

**Comment by ic005k** [comment]
> Are you planning to update the app or did you abandon it?

We will continue to update and maintain the software, of course.

You can download the required version of OCAT from the link below. I will be releasing OCAT in beta form until I have mastered the use of Unload; please follow the link provided below.

https://github.com/ic005k/OCAuxiliaryTools/actions

**Comment by 5T33Z0** [comment]
The problem is: "Unload" is an ARRAY. So in order to represent it in the GUI it needs a list in the UEFI section.

**Comment by ic005k** [comment]
> The problem is: "Unload" is an ARRAY. So in order to represent it in the GUI it needs a list in the UEFI section.

Yes, but I don't know how to use this Unload at the moment, so I don't display things like tables on the OCAT interface.

If you know how to use it, please give an example, I will refer to your example. (Since I haven't used a Hackintosh system in a long time, I can't test these things)

**Comment by 5T33Z0** [comment]
As far as I know, Unload is for unloading firmware drivers during boot, but requires analyzing the SysReport in order to create feasable entries:

![Bildschirmfoto 2024-10-10 um 06 21 42](https://github.com/user-attachments/assets/afc905c7-127d-4863-b5c8-a5e1ec206eb0)

I will play around with it on the weekend

**Comment by ic005k** [comment]
> As far as I know, Unload is for unloading firmware drivers during boot, but requires analyzing the SysReport in order to create feasable entries

Please download the latest OCAT development version in "Actions", I have made an intelligent prediction for Unload, you can test it.

For Unload, I think the OC official should include an example in "SampleCustom.plist" so that OCAT can adapt intelligently to it.

**Comment by 5T33Z0** [comment]
SampleCustom.plist currently does not contain any example entries in this array, I've checked it already

**Comment by NoReason0** [comment]
So how to download OCAT with Opencore 1.0.2? I run translate for first comment. And I download version 20240003 and latest file on actions but it is Opencore version 0.8.8. thanks.

**Comment by 5T33Z0** [comment]
> So how to download OCAT with Opencore 1.0.2? I run translate for first comment. And I download version 20240003 and latest file on actions but it is Opencore version 0.8.8. thanks.

I guess you meant "How to download OpenCore with OCAT"?

It's pretty simple: https://github.com/5T33Z0/OC-Little-Translated/tree/main/D_Updating_OpenCore

**Comment by NoReason0** [comment]
Sorry, I cant use English well. I mean my OCAT's ocvalidate is work on Opencore 0.8.8.

**Comment by 5T33Z0** [comment]
> Sorry, I cant use English well. I mean my OCAT's ocvalidate is work on Opencore 0.8.8.

1. Press Upgrade… <br>![uppgrade](https://github.com/user-attachments/assets/4ef9ac96-b37c-499a-b5f4-38ecf32d13d5)
2. <br>![release](https://github.com/user-attachments/assets/e3d6bef2-caaa-4c01-8cac-16771340750b)

**Comment by NoReason0** [comment]
Oh. I'm sorry. I think I dont find in detail.
